cards/event-standard/template.hbs

```diff
@@ -95,10 +97,11 @@
     {{#if ariaLabel}}aria-label="{{ariaLabel}}"{{/if}}>
     {{#if (any iconName iconUrl)}}
     <div class="HitchhikerCTA-iconWrapper">
-      <div class="HitchhikerCTA-icon" data-component="IconComponent" data-opts='{
-        "iconUrl": "{{#if iconUrl}}{{#unless (isNonRelativeUrl iconUrl)}}{{@root.relativePath}}/{{/unless}}{{iconUrl}}{{/if}}",
-        "iconName": "{{iconName}}"
-      }'>
+      <div class="HitchhikerCTA-icon">
+        {{> icons/iconPartial
+          iconName=iconName
+          iconUrl=(relativePathHandler url=iconUrl relativePath=@root.relativePath)
```

Comment:
What do you think about aligning the params with the partial name?

Reply:
+1, I like that stylistic convention.

Reply:
Will update! I tried looking for an automatic handlebars way to format handlebars like the "Prettier for Handlebars" extension, but it doesn't like our templates. You can't do just a single find and replace for partial invocations at different indent levels, so maybe we could look into something automatic in the future?
![image](https://user-images.githubusercontent.com/23005393/124620165-3cdb9580-de47-11eb-8d7a-4863f407f90f.png)

Reply:
Yeah, having something automatic in the future could be good.